.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+#### What problem is this solving?
+
+<!--- What is the motivation and context for this change? -->
+
+#### How should this be manually tested?
+
+[Workspace](url)
+
+#### Checklist/Reminders
+
+- [ ] Updated `README.md`.
+- [ ] Updated `CHANGELOG.md`.
+- [ ] Linked this PR to a Clubhouse story (if applicable).
+- [ ] Updated/created tests (important for bug fixes).
+- [ ] Deleted the workspace after merging this PR (if applicable).
```

Comment:
my opinion: I think we could remove this checklist, it seems to me that no one actually uses it and it's out of date (VTEX don't use Clubhouse anymore)